package.json

```diff
@@ -93,20 +90,27 @@
         }
     },
     "scripts": {
-        "clean": "rimraf out ./lsp/server/out ./lsp/client/out",
-        "compile": "rimraf ./lsp/server/tsconfig.tsbuildinfo ./lsp/client/tsconfig.tsbuildinfo && tsc -b ./ && npm run copy-resources",
-        "copy-resources": "cp -r './lsp/server/resources' './lsp/server/out/src/'",
+        "clean": "rimraf ./out",
+        "compile": "rimraf ./out/tsconfig.tsbuildinfo && tsc -b ./ && npm run copy-yaml",
+        "copy-yaml-win": "mkdir out\\src\\resources && copy resources\\component-experiences.yaml out\\src\\resources",
+        "copy-yaml-unix": "mkdir -p ./out/src/resources && cp ./resources/component-experiences.yaml ./out/src/resources",
+        "copy-yaml": "npm run copy-yaml-win || npm run copy-yaml-unix",
```

Comment:
This...is not my favorite, relying on fatal errors in the shell to take affirmative alternative action. It can be risky, and is otherwise not super intuitive. A more standard and maintainable way to do this would be to create a basic Node utility module to manage the logic. We know we have a Node.js environment in either case, so we can write a pretty simple OS-agnostic script with `fs`, `path`, and `os` to do the copying, give it an identifiable name, and voila, maintainable code. 🙂

It's slightly more code than this, but IMO it's easier to maintain, and more intuitive to understand.

Reply:
Let me take care in another PR. (This PR got merged few minutes ago.)

I think I remember why I had `yaml` folder under `resources` folder. It's because I was able to use `cp -r`. Since the yaml file was move and `cp` was failing on copying a single file claiming it can't find `resources` folder.